index.html

```diff
@@ -376,113 +396,203 @@ <h4>Characters</h4>
+            </li>
+            <li>an optional <a>On Screen</a> property, which is an annotation indicating the position of the <a>character</a> during the <a>Event</a></li>
+          </ul>
+        </p>
 
-        <section>
-          <h4>End</h4>
-          <p>Each <a>Event</a> MUST have a <dfn>End</dfn> property.</p>
-          <p class="note">Typically <a>Events</a> do not overlap in time. However, there may be cases where they do, e.g. when different <a>Characters</a> speak different text at the same time.</p>
+        <p>An <a>Event</a> is represented in TTML with the following structure and constraints:
+          <ul>There MUST be one <code>div</code> element, with the following constraints:
+            <ul>
+              <li>The <code>xml:id</code> attribute MUST be present containing the <a>Event Identifier</a>.</li>
+              <li>The <code>begin</code> and <code>end</code> attributes MUST be present and correspond to the <a>Start</a> and <a>End</a> of the <a>Event</a>.</li>
+              <li>The <code>ttm:agent</code> attribute MAY be present and if present, MUST contain a reference to the <code>ttm:agent</code> representing the <a>Character</a>.</li>
+          <pre class="example">
+...
+&lt;div xml:id=&quot;event_0130df43-5443-4206-b1d9-d331e42b5a86&quot;
+     begin=&quot;9663f&quot; end=&quot;9682f&quot; 
+     ttm:agent=&quot;character_bb89e9c6-4453-4402-a376-5a496cfb9839&quot;&gt;
+...
+&lt;/div&gt;
+...
+          </pre>
+              <li>It MUST contain one <code>p</code> element representing the <a>Main Text</a>.</li>
+              <li>It MAY contain another <code>p</code> element representing the <a>Contextual Text</a></li>
+              <li>The <code>style</code> attribute MAY be present. If present, it MAY contain a reference to the <code>style</code> defining the <a>Character Style</a>. Additional style references or inline styles MAY be used.</li>
+              <li>It MAY contain a <code>metadata</code> element representing the <a>On Screen</a> property.</li>
+          <pre class="example">
+...
+&lt;div xml:id=&quot;event_0130df43-5443-4206-b1d9-d331e42b5a86&quot;
+     begin=&quot;9663f&quot; end=&quot;9682f&quot; 
+     ttm:agent=&quot;character_bb89e9c6-4453-4402-a376-5a496cfb9839&quot;&gt;
+...
+&lt;/div&gt;
+...
+          </pre>
+            </ul>
+          </ul>
+        </p>
         </section>
 
         <section>
           <h4>Text</h4>
-          <p>Each <a>Event</a> MUST have a <dfn>Text</dfn> property, which provides the main text content for this <a>Event</a>.</p>
-          <p>It MAY have a <dfn>Text Styles</dfn> property, which provides styles associated with the <a>Text</a> of this <a>Event</a>.</p>
+          <p>The <dfn>Text</dfn> object is used in the <a>Main Text</a> property or the <a>Contextual Text</a> property of an <a>Event</a> object to provide text content in a single language, to document this language and provide optional styling for the text content.</p>
```

Comment:
Suggest wording like:

The Text object contains text content in a single language, and may be styled and associated with a Character. The Main Text and Contextual Text objects of Events are Text Objects.